t3/runners/rmg_runner.py

```diff
@@ -186,6 +186,18 @@ def run_rmg_incore(rmg_input_file_path: str,
     stdout, stderr = execute_command(commands, shell=True, no_fail=True, executable='/bin/bash')
     if 'RMG threw an exception and did not converge.\n' in stderr:
         return True
+        # Check for err file that can be generated by Julia
+    # Todo: Improve the error checking. This situation arose from Julia having a TypeError issue from a recent commit.
+    elif len(stderr)!=0:
+        print(stderr)
+    if os.path.isfile(os.path.join(project_directory, 'err.txt')):
+        with open(os.path.join(project_directory, 'err.txt'), 'r') as f:
+            lines = f.readlines()
+        for line_number, line in enumerate(lines):
+            print(line, line_number)
+        if 'Traceback (most recent call last):\n' in lines and 'Julia' in lines:
+            # Raise an error that will inform the user that Julia has failed
+            raise RuntimeError('Julia has raised an error with RMS and therefore RMG cannot complete. Please check your Julia installation\n')
```

Comment:
Is the desired behavior of T3 to raise an error here? Could be, I'm just double checking: If this problem happens, this means there's no way forward (maybe a few QM calcs will cause RMG to generate a different model and not to step on this Julia mine?)

Reply:
Yes, I think T3 should raise an error here because if Julia is broken then the RMG run will never work and thus the user will need to check their Julia environment/connection to see what the issue is